src/pages/graphql/schema/orders/mutations/confirm-cancel-order.md

```diff
@@ -0,0 +1,71 @@
+---
+title: confirmCancelOrder mutation
+---
+
+# confirmCancelOrder mutation
+
+The `confirmCancelOrder` mutation allows a guest to cancel an order, passing its encoded identifier and a confirmation key.
+
+A guest customer can cancel an order only if the following conditions are true:
+
+ * The guest has previously placed the order.
+ * The `order_cancellation_enabled` configuration setting is `true`.
+ * The order has a status of **RECEIVED**, **PENDING**, or **PROCESSING**.
+
+If the guest user has been charged for the order, the order is placed in the CLOSED state, and the user will be issued a refund. Otherwise, the status of the order is set to CANCELED.
+
```

Comment:
```suggestion
If the guest customer has been charged for the order, the order is placed in the CLOSED state, and the user will be issued a refund. Otherwise, the status of the order is set to CANCELED.

```